src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexFindOptimizations.cs

```diff
@@ -125,20 +122,16 @@ public RegexFindOptimizations(RegexNode root, RegexOptions options, CultureInfo
                         // The set contains one and only one character, meaning every match starts
                         // with the same literal value (potentially case-insensitive). Search for that.
                         FixedDistanceLiteral = (chars[0], 0);
-                        FindMode = set.CaseInsensitive ?
-                            FindNextStartingPositionMode.LeadingLiteral_RightToLeft_CaseInsensitive :
-                            FindNextStartingPositionMode.LeadingLiteral_RightToLeft_CaseSensitive;
+                        FindMode = FindNextStartingPositionMode.LeadingLiteral_RightToLeft_CaseSensitive;
```

Comment:
Can we remove all the _CaseSensitive / _CaseInsensitive suffixes from the enum members now?